### ISSUE TITLE
Cold Chain M stops receiving sensor logs after 4 days

### ISSUE THREAD

**Issue by adamdewey** [msupply-foundation/mobile#4433]
## Describe the bug

After 4 days of logging, Cold Chain M stops receiving logs from the BlueMaestro sensors. 

![image](https://user-images.githubusercontent.com/19318937/136503679-d7ded888-f796-49c1-8401-7f0be1fc5255.png)

![image](https://user-images.githubusercontent.com/19318937/136503696-ee3fd926-0ee1-4127-8f04-eb8cbe243e9c.png)

### To reproduce

To simulate a realistic environment for CCM, the screen was not left on and the app was logged into intermittently. However the tablet was on for the entire 4 days. 

Steps to reproduce the behaviour:

1. Connect a sensor to CCM. Leave the default settings.
2. Log into CCM once a day so that sensor data is sent to the tablet
3. Notice that after 4 days the sensor data is no longer received. 

*Note - 4 days is a rough estimate, your mileage may vary.

### Expected behaviour

Sensor data to continue to be received ad infinitum. 

### Proposed Solution

Leave if you don't know how to fix/implement. Edit this issue description and explain here if you know the best path of implementing the fix within the codebase.

### Version and device info

- App version: 8.3.0-rc1
- Tablet model: Lenovo TB-X505N
- OS version: Android 10

### Additional context

This problem is not seen when using Cold Chain SA.

**Comment by adamdewey** [comment]
Another occurrence of this, this time after only 1.5 days: 
![PXL_20220202_193638147](https://user-images.githubusercontent.com/19318937/152225139-3e680bf3-9730-4283-90b0-72b3b5f77a8d.jpg)

Also note this is on Android 11, not Android 10: 

App version: 8.4.1
Tablet model: Samsung Galaxy Tab A7 SM-T500
OS version: Android 11

**Comment by adamdewey** [comment]
A restart of the app fixed this, however
![PXL_20220202_194623079 MP](https://user-images.githubusercontent.com/19318937/152226767-889c3d3c-2913-4b14-b413-b0b31f3cec77.jpg)

**Comment by adamdewey** [comment]
🤔 wondering if that last occurrence on Android 11 was due to trying to reproduce the back button N/A bug (which then prevents temp logs from being downloaded from the sensor to the tablet). 

So that would potentially explain why an app restart fixed it?

**Comment by katherine-sussol** [comment]
Yeah, totally possible 🤔 . Do you happen to remember if an app restart fixed the original issue? I seem to recall it didn't, but am possible mis-remembering.

**Comment by adamdewey** [comment]
No, app start doesn't fix it on Android 10..
